xapian-letor/tests/api_letor.cc

```diff
@@ -104,26 +98,19 @@ gen_uniqterms_gt_doclen_db(Xapian::WritableDatabase& db, const string&)
     db.add_document(doc);
 }
 
-DEFINE_TESTCASE(ranker1, generated) {
+DEFINE_TESTCASE(listnet_ranker, generated) {
     Xapian::ListNETRanker ranker;
     string a = get_database_path("apitest_ranker1",
-				       gen_uniqterms_gt_doclen_db);
+				       populate_db);
```

Comment:
Wrapping after the comma's good, but in a parameter list the wrapped line should be indented to line up just after the `(` above.  This looks good in the latest version.